wolfcrypt/src/evp.c

```diff
@@ -792,6 +800,68 @@ static int wolfSSL_EVP_CipherUpdate_CCM(WOLFSSL_EVP_CIPHER_CTX *ctx,
 }
 #endif /* HAVE_AESCCM */
 
+#if defined(HAVE_ARIAGCM)
+static int wolfSSL_EVP_CipherUpdate_AriaGCM_AAD(WOLFSSL_EVP_CIPHER_CTX *ctx,
+        const unsigned char *in, int inl) {
+    if (in && inl > 0) {
+        byte* tmp = (byte*)XREALLOC(ctx->authIn,
```

Comment:
If possible can we avoid realloc's? Or if unavoidable, have the previous size known here when doing a realloc.

Reply:
This is basically a copy/paste of `wolfSSL_EVP_CipherUpdate_CCM_AAD` and `wolfSSL_EVP_CipherUpdate_GCM_AAD`.